stable/airflow/templates/_helpers.tpl

```diff
@@ -45,11 +45,31 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
+
 {{/*
-Create a default fully qualified redis cluster name or use the `redisHost` value if defined
+Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+NOTE: This is copied from the redis sub-chart and modified slightly:
 */}}
 {{- define "airflow.redis.fullname" -}}
```

Comment:
In the old version, this variable was deceptively named. It did not mirror what the Redis subchart called `fullname`, but rather mirrored what the subchart used as the created Service name. In the new version, `fullname` mirrors what the subchart does, and the host is set separately (see below).